commit e4660f93007e41c7dbfdb5e3e166c3a9e84a5ae6
Author: Igor Ivanov <igori@nvidia.com>
Date:   Thu Jun 17 16:07:31 2021 +0300

    issue: 2485156 Fix fc35 issues
    
    - glibc:
      The function pthread_yield has been deprecated; programs should use
      the equivalent standard function sched_yield instead.
    
    Signed-off-by: Igor Ivanov <igori@nvidia.com>

diff --git src/vma/iomux/epfd_info.cpp src/vma/iomux/epfd_info.cpp
index c7670c7b36b4..d7c6c4fcdf9e 100644
--- src/vma/iomux/epfd_info.cpp
+++ src/vma/iomux/epfd_info.cpp
@@ -624,7 +624,7 @@ int epfd_info::ring_poll_and_process_element(uint64_t *p_poll_sn, void* pv_fd_re
 
 	m_ring_map_lock.unlock();
 
-	if (m_sysvar_thread_mode == THREAD_MODE_PLENTY && ret_total == 0 && errno == EAGAIN) pthread_yield();
+	if (m_sysvar_thread_mode == THREAD_MODE_PLENTY && ret_total == 0 && errno == EAGAIN) sched_yield();
 
 	if (ret_total) {
 		__log_func("ret_total=%d", ret_total);
